bluebrain/repo-bluebrain/packages/sim-model/package.py

```diff
@@ -33,21 +33,18 @@ class ModelHippocampus(SimModel):
     homepage = ""
 
     variant('coreneuron',  default=False, description="Enable CoreNEURON Support")
-    variant('profile',     default=False, description="Enable profiling using Tau")
     variant('caliper',     default=False, description="Enable Caliper instrumentation")
 
     # neuron/corenrn get linked automatically when using nrnivmodl[-core]
     # Dont duplicate the link dependency (only 'build' and 'run')
     depends_on('neuron+mpi', type=('build', 'run'))
```

Comment:
Not something major but this implies that now by default `neurodamus` packages will depend on `coreneuron` also now. Maybe with this we want to remove `+coreneuron` in the deployed spec? Also this doesn't align with:
```
    variant('coreneuron',  default=False, description="Enable CoreNEURON Support")
``` 
in line 35 of this file. So if we don't specify `+coreneuron` installind the `neurodamus` models will always conflict?

Reply:
> Not something major but this implies that now by default neurodamus packages will depend on coreneuron also now. 

With the existing neuron releases (i.e. <= 8.99), it won't be by default because of

> depends_on('coreneuron', when='+coreneuron ^neuron@:8.99', type=('build', 'run'))

i.e. we are depending on coreneuron only when `+coreneuron` exist:

```console

$ spack spec -I neurodamus-hippocampus
Input spec
--------------------------------
 -   neurodamus-hippocampus

Concretized
--------------------------------
 -   neurodamus-hippocampus@1.8-2.12.9%gcc@11.2.0~caliper~coreneuron+mvdtool+synapsetool common_mods=default arch=linux-rhel7-skylake
[^]      ^gettext@0.21%gcc@11.2.0+bzip2+curses+git~libunistring+libxml2+tar+xz arch=linux-rhel7-skylake
[^]          ^bzip2@1.0.8%gcc@11.2.0~debug~pic+shared arch=linux-rhel7-skylake
[^]          ^libiconv@1.16%gcc@11.2.0 libs=shared,static arch=linux-rhel7-skylake
[^]          ^libxml2@2.9.12%gcc@11.2.0~python arch=linux-rhel7-skylake
[^]              ^pkgconf@1.8.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^xz@5.2.5%gcc@11.2.0~pic libs=shared,static arch=linux-rhel7-skylake
[^]              ^zlib@1.2.11%gcc@11.2.0+optimize+pic+shared arch=linux-rhel7-skylake
[^]          ^ncurses@6.2%gcc@11.2.0~symlinks+termlib abi=none arch=linux-rhel7-skylake
[^]          ^tar@1.28%gcc@11.2.0 patches=08921fcbd732050c74ddf1de7d8ad95ffdbc09f8b4342456fa2f6a0dd02a957c,125cd6142fac2cc339e9aebfe79e40f90766022b8e8401532b1729e84fc148c2,5c314db58d005043bf407abaf25eb9823b9032a22fd12a0b142d4bf548130fa4,d428578be7fb99b831eb61e53b8d88a859afe08b479a21238180899707d79ce4 arch=linux-rhel7-skylake
[^]      ^hdf5@1.10.7%gcc@11.2.0+cxx~fortran+hl~ipo~java+mpi+shared~szip~threadsafe+tools api=default build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^cmake@3.21.4%gcc@11.2.0~doc+ncurses+openssl+ownlibs~qt build_type=Release arch=linux-rhel7-skylake
[^]          ^hpe-mpi@2.25.hmpt%gcc@11.2.0 arch=linux-rhel7-skylake
[^]      ^libsonata-report@1.1.1%gcc@11.2.0~ipo+mpi build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^spdlog@1.9.2%gcc@11.2.0~ipo+shared build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^fmt@9.1.0%gcc@11.2.0~ipo+pic~shared build_type=RelWithDebInfo cxxstd=11 arch=linux-rhel7-skylake
 -       ^neuron@8.2.2a%gcc@11.2.0+binary~caliper~codegenopt+coreneuron~debug~gpu~interviews~ipo+legacy-fr+legacy-unit+memacs+mod-compatibility+mpi~nmodl~openmp+python+report+rx3d+shared~sympy~sympyopt~tests~unified build_type=RelWithDebInfo model_tests=None patches=708cb04826b394a858069d93e8c08e1e81e914c23e1ef3da0486e8233834ff6c sanitizers=None arch=linux-rhel7-skylake
 -           ^bison@3.8.2%gcc@11.2.0 arch=linux-rhel7-skylake
 -           ^coreneuron@8.2.3.20221125%gcc@11.2.0~caliper~codegenopt~gpu~ipo~knl+legacy-unit+mpi~nmodl~openmp~profile+report+shared~sympy~sympyopt~tests~unified build_type=RelWithDebInfo arch=linux-rhel7-skylake
 -               ^flex@2.6.3%gcc@11.2.0+lex~nls arch=linux-rhel7-skylake
 -               ^ninja@1.10.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^python@3.9.7%gcc@11.2.0+bz2+ctypes+dbm~debug+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tix~tkinter~ucs4+uuid+zlib patches=0d98e93189bc278fbc37a50ed7f183bd8aaf249a8e1670a465f0db6bb4f8cf87,4c2457325f2b608b1b6a2c63087df8c26e07db3e3d493caf36a56f0ecf6fb768,f2fd060afc4b4618fe8104c4c5d771f36dc55b1db5a4623785a4ea707ec72fb4 arch=linux-rhel7-skylake
[^]              ^reportinglib@2.5.6%gcc@11.2.0~ipo~profile+shared~tests build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^py-cython@0.29.24%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-setuptools@57.4.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-numpy@1.19.5%gcc@11.2.0+blas+lapack patches=8a9d5d1b3f145c043b8b04869e7d46c6ff95c3f486d84f69693017c7e6190c7d arch=linux-rhel7-skylake
[^]              ^intel-mkl@2020.4.304%gcc@11.2.0~ilp64+shared threads=none arch=linux-rhel7-skylake
[^]          ^readline@8.1%gcc@11.2.0 arch=linux-rhel7-skylake
[^]      ^py-mvdtool@2.4.2%gcc@11.2.0+mpi arch=linux-rhel7-skylake
[^]          ^boost@1.79.0%gcc@11.2.0+atomic+chrono~clanglibcpp~container~context~coroutine+date_time~debug+exception~fiber+filesystem+graph~icu+iostreams+locale+log+math~mpi+multithreaded~numpy+pic+program_options~python+random+regex+serialization+shared+signals~singlethreaded+system~taggedlayout+test+thread+timer~versionedlayout+wave cxxstd=98 visibility=hidden arch=linux-rhel7-skylake
[^]          ^highfive@2.6.2%gcc@11.2.0+boost~eigen~ipo+mpi~xtensor build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^libsonata@0.1.16%gcc@11.2.0~ipo+mpi build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^catch2@3.0.1%gcc@11.2.0~ipo build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^nlohmann-json@3.9.1%gcc@11.2.0~ipo~multiple_headers build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^py-setuptools-scm@6.3.2%gcc@11.2.0+toml arch=linux-rhel7-skylake
[^]                  ^py-packaging@21.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                      ^py-pyparsing@2.4.7%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                  ^py-tomli@1.2.1%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                      ^py-pip@21.1.2%gcc@11.2.0 arch=linux-rhel7-skylake
 -       ^py-neurodamus@2.12.9%gcc@11.2.0~all_deps arch=linux-rhel7-skylake
[^]          ^py-docopt@0.6.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-h5py@3.4.0%gcc@11.2.0+mpi arch=linux-rhel7-skylake
[^]              ^py-mpi4py@3.1.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-pkgconfig@1.5.1%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-wheel@0.36.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-libsonata@0.1.16%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-pybind11@2.9.1%gcc@11.2.0~ipo build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^py-morphio@3.3.3%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-psutil@5.8.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]      ^synapsetool@0.6.3%gcc@11.2.0~ipo+mpi~python+shared build_type=RelWithDebInfo arch=linux-rhel7-skylake
```

and

```console
$ spack spec -I neurodamus-hippocampus+coreneuron
Input spec
--------------------------------
 -   neurodamus-hippocampus+coreneuron

Concretized
--------------------------------
 -   neurodamus-hippocampus@1.8-2.12.9%gcc@11.2.0~caliper+coreneuron+mvdtool+synapsetool common_mods=default arch=linux-rhel7-skylake
 -       ^coreneuron@8.2.3.20221125%gcc@11.2.0~caliper~codegenopt~gpu~ipo~knl+legacy-unit+mpi~nmodl~openmp~profile+report+shared~sympy~sympyopt~tests~unified build_type=RelWithDebInfo arch=linux-rhel7-skylake
 -           ^bison@3.8.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^cmake@3.21.4%gcc@11.2.0~doc+ncurses+openssl+ownlibs~qt build_type=Release arch=linux-rhel7-skylake
 -           ^flex@2.6.3%gcc@11.2.0+lex~nls arch=linux-rhel7-skylake
[^]          ^hpe-mpi@2.25.hmpt%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^libsonata-report@1.1.1%gcc@11.2.0~ipo+mpi build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^hdf5@1.10.7%gcc@11.2.0+cxx~fortran+hl~ipo~java+mpi+shared~szip~threadsafe+tools api=default build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]                  ^pkgconf@1.8.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                  ^zlib@1.2.11%gcc@11.2.0+optimize+pic+shared arch=linux-rhel7-skylake
[^]              ^spdlog@1.9.2%gcc@11.2.0~ipo+shared build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]                  ^fmt@9.1.0%gcc@11.2.0~ipo+pic~shared build_type=RelWithDebInfo cxxstd=11 arch=linux-rhel7-skylake
 -           ^ninja@1.10.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^python@3.9.7%gcc@11.2.0+bz2+ctypes+dbm~debug+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tix~tkinter~ucs4+uuid+zlib patches=0d98e93189bc278fbc37a50ed7f183bd8aaf249a8e1670a465f0db6bb4f8cf87,4c2457325f2b608b1b6a2c63087df8c26e07db3e3d493caf36a56f0ecf6fb768,f2fd060afc4b4618fe8104c4c5d771f36dc55b1db5a4623785a4ea707ec72fb4 arch=linux-rhel7-skylake
[^]          ^reportinglib@2.5.6%gcc@11.2.0~ipo~profile+shared~tests build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]      ^gettext@0.21%gcc@11.2.0+bzip2+curses+git~libunistring+libxml2+tar+xz arch=linux-rhel7-skylake
[^]          ^bzip2@1.0.8%gcc@11.2.0~debug~pic+shared arch=linux-rhel7-skylake
[^]          ^libiconv@1.16%gcc@11.2.0 libs=shared,static arch=linux-rhel7-skylake
[^]          ^libxml2@2.9.12%gcc@11.2.0~python arch=linux-rhel7-skylake
[^]              ^xz@5.2.5%gcc@11.2.0~pic libs=shared,static arch=linux-rhel7-skylake
[^]          ^ncurses@6.2%gcc@11.2.0~symlinks+termlib abi=none arch=linux-rhel7-skylake
[^]          ^tar@1.28%gcc@11.2.0 patches=08921fcbd732050c74ddf1de7d8ad95ffdbc09f8b4342456fa2f6a0dd02a957c,125cd6142fac2cc339e9aebfe79e40f90766022b8e8401532b1729e84fc148c2,5c314db58d005043bf407abaf25eb9823b9032a22fd12a0b142d4bf548130fa4,d428578be7fb99b831eb61e53b8d88a859afe08b479a21238180899707d79ce4 arch=linux-rhel7-skylake
 -       ^neuron@8.2.2a%gcc@11.2.0+binary~caliper~codegenopt+coreneuron~debug~gpu~interviews~ipo+legacy-fr+legacy-unit+memacs+mod-compatibility+mpi~nmodl~openmp+python+report+rx3d+shared~sympy~sympyopt~tests~unified build_type=RelWithDebInfo model_tests=None patches=708cb04826b394a858069d93e8c08e1e81e914c23e1ef3da0486e8233834ff6c sanitizers=None arch=linux-rhel7-skylake
[^]          ^py-cython@0.29.24%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-setuptools@57.4.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-numpy@1.19.5%gcc@11.2.0+blas+lapack patches=8a9d5d1b3f145c043b8b04869e7d46c6ff95c3f486d84f69693017c7e6190c7d arch=linux-rhel7-skylake
[^]              ^intel-mkl@2020.4.304%gcc@11.2.0~ilp64+shared threads=none arch=linux-rhel7-skylake
[^]          ^readline@8.1%gcc@11.2.0 arch=linux-rhel7-skylake
[^]      ^py-mvdtool@2.4.2%gcc@11.2.0+mpi arch=linux-rhel7-skylake
[^]          ^boost@1.79.0%gcc@11.2.0+atomic+chrono~clanglibcpp~container~context~coroutine+date_time~debug+exception~fiber+filesystem+graph~icu+iostreams+locale+log+math~mpi+multithreaded~numpy+pic+program_options~python+random+regex+serialization+shared+signals~singlethreaded+system~taggedlayout+test+thread+timer~versionedlayout+wave cxxstd=98 visibility=hidden arch=linux-rhel7-skylake
[^]          ^highfive@2.6.2%gcc@11.2.0+boost~eigen~ipo+mpi~xtensor build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^libsonata@0.1.16%gcc@11.2.0~ipo+mpi build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^catch2@3.0.1%gcc@11.2.0~ipo build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^nlohmann-json@3.9.1%gcc@11.2.0~ipo~multiple_headers build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^py-setuptools-scm@6.3.2%gcc@11.2.0+toml arch=linux-rhel7-skylake
[^]                  ^py-packaging@21.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                      ^py-pyparsing@2.4.7%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                  ^py-tomli@1.2.1%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                      ^py-pip@21.1.2%gcc@11.2.0 arch=linux-rhel7-skylake
 -       ^py-neurodamus@2.12.9%gcc@11.2.0~all_deps arch=linux-rhel7-skylake
[^]          ^py-docopt@0.6.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-h5py@3.4.0%gcc@11.2.0+mpi arch=linux-rhel7-skylake
[^]              ^py-mpi4py@3.1.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-pkgconfig@1.5.1%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-wheel@0.36.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-libsonata@0.1.16%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-pybind11@2.9.1%gcc@11.2.0~ipo build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^py-morphio@3.3.3%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-psutil@5.8.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]      ^synapsetool@0.6.3%gcc@11.2.0~ipo+mpi~python+shared build_type=RelWithDebInfo arch=linux-rhel7-skylake
```

And with `neuron@develop`, it's true that coreneuron will be by default shipped inside the neuron but I didn't want to disable that in order to avoid multiple neuron installations. In the future for simplicity, we are assuming there is a neuron which always has coreneuron enabled :

```console
$ spack spec -I neurodamus-hippocampus ^neuron@develop
Input spec
--------------------------------
 -   neurodamus-hippocampus
 -       ^neuron@develop

Concretized
--------------------------------
 -   neurodamus-hippocampus@1.8-2.12.9%gcc@11.2.0~caliper~coreneuron+mvdtool+synapsetool common_mods=default arch=linux-rhel7-skylake
[^]      ^gettext@0.21%gcc@11.2.0+bzip2+curses+git~libunistring+libxml2+tar+xz arch=linux-rhel7-skylake
[^]          ^bzip2@1.0.8%gcc@11.2.0~debug~pic+shared arch=linux-rhel7-skylake
[^]          ^libiconv@1.16%gcc@11.2.0 libs=shared,static arch=linux-rhel7-skylake
[^]          ^libxml2@2.9.12%gcc@11.2.0~python arch=linux-rhel7-skylake
[^]              ^pkgconf@1.8.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^xz@5.2.5%gcc@11.2.0~pic libs=shared,static arch=linux-rhel7-skylake
[^]              ^zlib@1.2.11%gcc@11.2.0+optimize+pic+shared arch=linux-rhel7-skylake
[^]          ^ncurses@6.2%gcc@11.2.0~symlinks+termlib abi=none arch=linux-rhel7-skylake
[^]          ^tar@1.28%gcc@11.2.0 patches=08921fcbd732050c74ddf1de7d8ad95ffdbc09f8b4342456fa2f6a0dd02a957c,125cd6142fac2cc339e9aebfe79e40f90766022b8e8401532b1729e84fc148c2,5c314db58d005043bf407abaf25eb9823b9032a22fd12a0b142d4bf548130fa4,d428578be7fb99b831eb61e53b8d88a859afe08b479a21238180899707d79ce4 arch=linux-rhel7-skylake
[^]      ^hdf5@1.10.7%gcc@11.2.0+cxx~fortran+hl~ipo~java+mpi+shared~szip~threadsafe+tools api=default build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^cmake@3.21.4%gcc@11.2.0~doc+ncurses+openssl+ownlibs~qt build_type=Release arch=linux-rhel7-skylake
[^]          ^hpe-mpi@2.25.hmpt%gcc@11.2.0 arch=linux-rhel7-skylake
[^]      ^libsonata-report@1.1.1%gcc@11.2.0~ipo+mpi build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^spdlog@1.9.2%gcc@11.2.0~ipo+shared build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^fmt@9.1.0%gcc@11.2.0~ipo+pic~shared build_type=RelWithDebInfo cxxstd=11 arch=linux-rhel7-skylake
 -       ^neuron@develop%gcc@11.2.0+binary~caliper~codegenopt+coreneuron~debug~gpu~interviews~ipo+legacy-fr+legacy-unit+memacs+mod-compatibility+mpi~nmodl~openmp+python+report+rx3d+shared~sympy~sympyopt~tests~unified build_type=RelWithDebInfo model_tests=None patches=708cb04826b394a858069d93e8c08e1e81e914c23e1ef3da0486e8233834ff6c sanitizers=None arch=linux-rhel7-skylake
 -           ^bison@3.8.2%gcc@11.2.0 arch=linux-rhel7-skylake
 -           ^flex@2.6.3%gcc@11.2.0+lex~nls arch=linux-rhel7-skylake
 -           ^ninja@1.10.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-cython@0.29.24%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-setuptools@57.4.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                  ^python@3.9.7%gcc@11.2.0+bz2+ctypes+dbm~debug+libxml2+lzma~nis~optimizations+pic+pyexpat+pythoncmd+readline+shared+sqlite3+ssl~tix~tkinter~ucs4+uuid+zlib patches=0d98e93189bc278fbc37a50ed7f183bd8aaf249a8e1670a465f0db6bb4f8cf87,4c2457325f2b608b1b6a2c63087df8c26e07db3e3d493caf36a56f0ecf6fb768,f2fd060afc4b4618fe8104c4c5d771f36dc55b1db5a4623785a4ea707ec72fb4 arch=linux-rhel7-skylake
[^]          ^py-numpy@1.19.5%gcc@11.2.0+blas+lapack patches=8a9d5d1b3f145c043b8b04869e7d46c6ff95c3f486d84f69693017c7e6190c7d arch=linux-rhel7-skylake
[^]              ^intel-mkl@2020.4.304%gcc@11.2.0~ilp64+shared threads=none arch=linux-rhel7-skylake
[^]          ^readline@8.1%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^reportinglib@2.5.6%gcc@11.2.0~ipo~profile+shared~tests build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]      ^py-mvdtool@2.4.2%gcc@11.2.0+mpi arch=linux-rhel7-skylake
[^]          ^boost@1.79.0%gcc@11.2.0+atomic+chrono~clanglibcpp~container~context~coroutine+date_time~debug+exception~fiber+filesystem+graph~icu+iostreams+locale+log+math~mpi+multithreaded~numpy+pic+program_options~python+random+regex+serialization+shared+signals~singlethreaded+system~taggedlayout+test+thread+timer~versionedlayout+wave cxxstd=98 visibility=hidden arch=linux-rhel7-skylake
[^]          ^highfive@2.6.2%gcc@11.2.0+boost~eigen~ipo+mpi~xtensor build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^libsonata@0.1.16%gcc@11.2.0~ipo+mpi build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^catch2@3.0.1%gcc@11.2.0~ipo build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^nlohmann-json@3.9.1%gcc@11.2.0~ipo~multiple_headers build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]              ^py-setuptools-scm@6.3.2%gcc@11.2.0+toml arch=linux-rhel7-skylake
[^]                  ^py-packaging@21.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                      ^py-pyparsing@2.4.7%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                  ^py-tomli@1.2.1%gcc@11.2.0 arch=linux-rhel7-skylake
[^]                      ^py-pip@21.1.2%gcc@11.2.0 arch=linux-rhel7-skylake
 -       ^py-neurodamus@2.12.9%gcc@11.2.0~all_deps arch=linux-rhel7-skylake
[^]          ^py-docopt@0.6.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-h5py@3.4.0%gcc@11.2.0+mpi arch=linux-rhel7-skylake
[^]              ^py-mpi4py@3.1.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-pkgconfig@1.5.1%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-wheel@0.36.2%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-libsonata@0.1.16%gcc@11.2.0 arch=linux-rhel7-skylake
[^]              ^py-pybind11@2.9.1%gcc@11.2.0~ipo build_type=RelWithDebInfo arch=linux-rhel7-skylake
[^]          ^py-morphio@3.3.3%gcc@11.2.0 arch=linux-rhel7-skylake
[^]          ^py-psutil@5.8.0%gcc@11.2.0 arch=linux-rhel7-skylake
[^]      ^synapsetool@0.6.3%gcc@11.2.0~ipo+mpi~python+shared build_type=RelWithDebInfo arch=linux-rhel7-skylake
```

Let me know if I have missed anything obvious!

Reply:
Sounds right to me 👍 Thanks for the clarifications